edge-functions/add-header/README.md

```diff
@@ -13,7 +13,7 @@ demoUrl: https://edge-functions-add-header.vercel.app
 
 # Add Header Example
 
-Below is the code from [middleware.ts](middleware.ts) showing how to add headers at the edge:
+Below is the code from [middleware.ts](middleware.ts) showing how to add response headers at the edge:
```

Comment:
```suggestion
Below is the code from [middleware.ts](middleware.ts) showing how to add response headers at the edge (available since Next.js v13.0.0):
```